Cython/Utility/CppInSTL.cpp

```diff
@@ -0,0 +1,70 @@
+//////////////////// InContainers.proto ////////////////////
+
+#include <algorithm>
+
+namespace __operator_in_containers_helpers__ {
```

Comment:
We prefix all internal user visible names with `__pyx_` in order to avoid conflicts.